api/lib/mno_enterprise/concerns/controllers/jpi/v1/organizations_controller.rb

```diff
@@ -168,6 +168,17 @@ def remove_member
     render 'members'
   end
 
+
+    # POST /mnoe/jpi/v1/organizations/:id/freeze
+    def halt
+      # Check the password provided
+      return head :forbidden unless current_user.class.authenticate_user({email: current_user.email, password: params[:password]})
```

Comment:
You can user `current_user.valid_password?(params.require(:password))` instead

Reply:
`valid_password?` is defined by `Devise::Models::DatabaseAuthenticatable` which cannot be used here. It could be added in `remote_authenticatable` via a custom endpoint but I'm not sure it is necessary